content/install-pop.md

```diff
@@ -87,3 +87,5 @@ If you installed from the Intel/AMD ISO and your system has discrete NVIDIA grap
 ```
 sudo apt install system76-driver-nvidia
 ```
+
```

Comment:
Please format as:

**NOTE:** If you used the ISO with NVIDIA drivers included, this step is not required.


You may also need to rebase to make sure the branch is up to date. :)